ovh/resource_ovh_cloud_project_kube.go

```diff
@@ -43,6 +43,11 @@ func resourceCloudProjectKube() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"privateNetworkId": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
are you sure it's forcenew ? (if there's a change, it should destroy/create the cluster ?)

Reply:
Yes, can't be changed afterwards